<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>781</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Обмен валют</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="fromLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Из:</string>
    </property>
   </widget>
   <widget class="QLabel" name="toLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>В:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="currencyFromComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>USD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RUB</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="currencyToComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>USD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RUB</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="amountLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="exchangeRatesLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>171</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Курс валют:
USD/RUB 67.76
EUR/RUB 73.42
EUR/USD 1.08</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Обменять</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitPushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
